### DISCUSSION TITLE
Can't use any AppImage: cannot execute: required file not found

### DISCUSSION THREAD

**Discussion by schnittchen** [TheAssassin/AppImageLauncher#655] [Q&A] [answered]
I first installed the lite version, but 80% of the time ArduinoIDE would not launch, while I could still launch it from the console without problem.

I then uninstalled the lite version and installed the rpm. Same problem.

So I decided to `ail-cli unintegrate path_to_appimage` and then uninstall AppImageLauncher. The result is that I cannot launch ArduinoIDE at all, even though it's the exact same download that worked before I touched AppImageLauncher.

Here is what I get:
```
 ~/Appimage ✓ ./arduino-ide_2.3.2_Linux_64bit.AppImage --no-sandbox
bash: ./arduino-ide_2.3.2_Linux_64bit.AppImage: cannot execute: required file not found
```

In fact, when downloading a different Appimage and chmoding it, I get the same error.

There must be something on my system that AppImageLauncher altered, and I need it to be reset so I can work again

**Comment by TheAssassin** [comment, marked as answer]
Please reboot. That should clean up the binfmt_misc integration. Alternatively, you can try to `sudo systemctl restart systemd-binfmt` first. Please report your results. RPM distros are a bit annoying when it comes to uninstalling, hence the uninstall hook asks for a reboot.

**Reply by schnittchen** [reply]
`sudo systemctl restart systemd-binfmt` was not sufficient. Works after a reboot. Thank you!

**Comment by Optiligence** [comment]
i get this error in a clean chroot (trying to extract the appimage contents)

**Reply by TheAssassin** [reply]
Please try to install the latest 3.0.0 alpha release from https://github.com/TheAssassin/AppImageLauncher/releases/.
